### DISCUSSION TITLE
Duplicate Check changed?

### DISCUSSION THREAD

**Discussion by bartekd123** [firefly-iii/firefly-iii#4679] [Firefly III]
Im still looking into it on my system, but has anything changed with how the duplicates are treated in the last couple updates? i did an update to the latest, and ran an import. When I run imports I usually import all from the beginning of the month. After doing the update, it looks like any transactions I updated manually after they were created in the previous version, and run a csvimport, it is adding the transactions again.

Im still looking at it to see if its something with my data, but thought Id ask if anyone else is running in to this.

**Comment by JC5** [comment]
I'm not aware myself of any code changes in this regard. If you run into it, check out the [debug view](https://docs.firefly-iii.org/csv/errors/duplicate_transactions/) to see what's different in certain transactions.

**Comment by bartekd123** [comment]
THanks for the reply. I ended up going through all my april transactions, updating the new one created to match the one imported earlier, and deleting the one imported earlier in the month. So I am good. but out of curiosity, I saw your message after, and imported a few from march which created duplicates. Looking at one, I tried to compare and don't see much difference, apart from ID's import dates, etc. There are additional fields that aren't in the earlier one.

I am good for now, since I won't be importing previous transactions, but can post the 2 JSON's if it is helpful for you at all, or you are curious?

**Reply by JC5** [reply]
Absolutely, or drop it in my mail (james@firefly-iii.org). I may see nothing at all, but you never know!

**Reply by bartekd123** [reply]
Thanks. I emailed it to you. Have a good night!

**Reply by JC5** [reply]
Thanks! I got it. I can't detect the difference though, the difference I see should have been ignored :(

Sorry about the mixup though.

**Reply by bartekd123** [reply]
No problem at all. It was only like 30 transactions that was able to find easily, and delete the old ones & modify the new ones so I am back on track. Thanks for taking a look. :)
